js/app.js

```diff
@@ -87,6 +87,16 @@ controller('PastMeetupsController', function($scope) {
     return meetupDate > now;
   };
   $scope.MeetupList = [
+    {
+      Name: "Availability and monitoring / alerting",
```

Comment:
To keep this consistent with https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/README.md ommit the spaces around the forward slash.

Reply:
Deliberately added the spaces around the slash because the text was displayed in a single line overflowing the UI's container box.

Reply:
Got it! Thanks!